app/mailer/server/api.js

```diff
@@ -50,7 +50,13 @@ export const replaceEscaped = (str, data = {}) => replace(str, {
 		return ret;
 	}, {}),
 });
-export const wrap = (html, data = {}) => replaceEscaped(body.replace('{{body}}', html), data);
+export const wrap = (html, data = {}) => {
+	if (settings.get('email_plain_text_only')) {
+		return replace(html, data);
```

Comment:
this should also replace `{{body}}` with email content right?

Reply:
No because here we do not use the body variable used below